### DISCUSSION TITLE
Issue with STL placements in new Model

### DISCUSSION THREAD

**Discussion by AkashVardhan7** [google-deepmind/mujoco#1983] [Asking for Help]
Hi I am Researcher at Georgia Tech, trying to build a simulation of 3 link robots in Mujoco. I have been able to create .obj files from my stl meshes which I then converted to a simple xml file which is being used to run the simulation. However when I load the links 1 by 1 to build up the assembly I find that the free joint I am creating to define the body's movement with the floor is being placed at a location which is offset from the center of the link containing the joint. How do I have the joint be located at the center of the central link. 
Find attached a snapshot of my simulation scene with the body frames for the floor and the central link with the joint.
![BottomUp3](https://github.com/google-deepmind/mujoco/assets/48886152/dc971f1b-e6aa-4bc0-bab1-8717867e3375)
Here's my XML file.
<mujoco model="smarticle">
    <compiler angle="radian" coordinate="local"/>
    <option timestep="0.001" gravity="0 0 0"/>

    <asset>
        <material name="MatPlane" rgba="0.8 0.8 0.8 1"/>
        <material name="RedMat" rgba="1 0 0 1"/>
        <material name="BlackMat" rgba="0 0 0 1"/>
        <mesh name="CentralLinkMesh" file="Central_Link.obj"/>
        <mesh name="LeftWingMesh" file="Wing3.obj"/>
        <mesh name="RightWingMesh" file="Wing4.obj"/>
    </asset>

    <worldbody>
        
        <body name="floor" pos="0 0 0">
            <geom name="floor" type="plane" size="10 10 0.1" material="MatPlane" condim="3" friction="1.0 0.005 0.0001"/>
        </body>

        
        <body name="central_link" pos="0 0 0.025" euler="0 0 0">
            <joint name="free_joint" type="free" pos="-0.085 0 0.0125"/>
            <geom type="mesh" mesh="CentralLinkMesh" material="RedMat" pos="-0.085 0 0.0125" condim="3" friction="5 0.005 0.0001" solref="0.02 1" solimp="0.8 0.9 0.01"/>
        </body>
    </worldbody>
</mujoco>

Thanks for your support.

**Comment by yuvaltassa** [comment]
https://mujoco.readthedocs.io/en/latest/XMLreference.html#body-freejoint-align
